_episodes/07-functions.md

```diff
@@ -217,14 +217,14 @@ DAT_FILES=$(patsubst books/%.txt, %.dat, $(TXT_FILES))
 
 # Generate summary table.
 results.txt : $(DAT_FILES) $(ZIPF_SRC)
-	$(ZIPF_EXE) $< > $@
+	$(ZIPF_EXE) $(DAT_FILES) > $@
 
 # Count words.
 .PHONY : dats
 dats : $(DAT_FILES)
 
 %.dat : books/%.txt $(COUNT_SRC)
-	$(COUNT_EXE) $< $*.dat
+	$(COUNT_EXE) $< $@
```

Comment:
This has been "silently" changed, and so is easy to miss. We should explicitly say what we're doing here. I suggest inserting at line 140:

> Let's also tidy up the %.dat rule using the automatic variable `$@` instead of `$*.dat`:
> ```
> %.dat : books/%.txt $(COUNT_SRC)
> 	$(COUNT_EXE) $< $@
> ```
> {: .make}